quantilizer.py

```diff
@@ -8,122 +8,225 @@
 import tensorflow as tf
 import tempfile
 import argparse
-from sklearn.neural_network import MLPClassifier
-from sklearn.linear_model import LogisticRegression
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import OneHotEncoder
 from helpers import graph_one, graph_two
+import keras
+from keras.models import Sequential
+from keras.layers import Dense, Dropout, Activation
+from keras.optimizers import SGD
+from keras import callbacks
+from datetime import datetime
+import time
+from wrappers import RobustRewardEnv
+import os
 
 def traj_segment_generator(pi, env, horizon, play=False):
+
```

Comment:
I was using four spaces for a tab in all of my code, so ideally we would keep that consistent.

Reply:
My preferences in VSCode and vim are "insert 4 spaces when tab". Don't know why but now there seems to be tabs everywhere.